Automattic/wp-calypso
# Settings: Better handling of Podcasting settings on private sites

The Podcasting category feed needs to be publicly accessible in order for podcasting to work, but this is not the case for private sites.

So, for private sites, this PR shows a more informative message instead of linking to the podcasting settings screen:

> ![2018-06-01t12 48 15-0500](https://user-images.githubusercontent.com/227022/40856564-f6290ba6-659d-11e8-8b74-f3c08c222c20.png)

If a user does somehow end up on the podcasting settings page for a private site, we can show the same message:

> ![2018-06-01t13 04 47-0500](https://user-images.githubusercontent.com/227022/40856588-00e99e52-659e-11e8-83b5-b8dff6400f00.png)

### To test

Enable the `manage/site-settings/podcasting` feature flag (on by default in development).

For a private site, verify that the first message above is shown on the Settings > Writing screen instead of a link to manage podcasting settings.

For a private site, visit the podcasting settings page `/settings/podcasting/:site` and verify that the second message above is shown.

Verify that clicking the link in each message takes you to the Privacy section of the Settings > General screen.
